tensorflow_quantum/core/proto/BUILD

```diff
@@ -1,11 +1,9 @@
+load("@com_google_protobuf//:protobuf.bzl", "py_proto_library")
+
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
-load("@com_google_protobuf//:protobuf.bzl", "py_proto_library")
-
-# load("//tools/build_defs/proto/cpp:cc_proto_library.bzl", "cc_proto_library")
```

Comment:
Should we remove this ?

Reply:
Oops, nice catch. it should be removed.

Reply:
I mean, the comment should be removed because it's internal. Also, the public rule at the line 5 should be moved to the line 1 due to internal BUILD file linter.